sql/2022/page-weight/facades-usage.sql

```diff
@@ -0,0 +1,20 @@
+SELECT 
+    _TABLE_SUFFIX as client,
+    COUNTIF(
+        REGEXP_CONTAINS(report,r'third-party-facades\/\).","score":null')
+    ) AS null_count,
+    COUNTIF(
+        REGEXP_CONTAINS(report,r'third-party-facades\/\).","score":1')
+    ) AS pass_count,
+    COUNTIF(
+        REGEXP_CONTAINS(report,r'third-party-facades\/\).","score":0')
+    ) AS fail_count

```

Comment:
These `REGEXP_CONTAINS` checks actually work fine, but we should use the semantic JSON functions built into BigQuery. For example:

```suggestion
    COUNTIF(JSON_VALUE(report, '$.audits.third-party-facades.score') IS NULL) AS null_count,
    COUNTIF(SAFE_CAST(JSON_VALUE(report, '$.audits.third-party-facades.score') AS FLOAT64) >= 0.9) AS pass_count,
    COUNTIF(SAFE_CAST(JSON_VALUE(report, '$.audits.third-party-facades.score') AS FLOAT64) < 0.9) AS fail_count
```

This also lets you use 0.9 as the pass/fail threshold, which is more consistent with the way Lighthouse scores audits. In this particular case everything is 0 or null, but it generalizes to all audits.